### PR TITLE
fix Typescript import syntax (fixes #47)

### LINKED ISSUE #47: inplace is undefined
I just installed jest-ratchet and I'm getting the following error.

```
Error: An error occurred while adding the reporter at path "/code/omega/node_modules/jest-ratchet/index.js"._inplace is not defined
    at reporters.forEach (/code/omega/node_modules/jest/node_modules/jest-cli/build/TestScheduler.js:510:15)
    at Array.forEach (<anonymous>)
    at TestScheduler._addCustomReporters (/code/omega/node_modules/jest/node_modules/jest-cli/build/TestScheduler.js:497:15)
    at TestScheduler._setupReporters (/code/omega/node_modules/jest/node_modules/jest-cli/build/TestScheduler.js:466:12)
    at new TestScheduler (/code/omega/node_modules/jest/node_modules/jest-cli/build/TestScheduler.js:193:10)
    at /code/omega/node_modules/jest/node_modules/jest-cli/build/runJest.js:452:27
    at Generator.next (<anonymous>)
    at step (/code/omega/node_modules/jest/node_modules/jest-cli/build/runJest.js:107:30)
    at /code/omega/node_modules/jest/node_modules/jest-cli/build/runJest.js:118:15
```

The source is this line in index.js.

```js
var inplace = _inplace;
```

It seems to come down to the fact that `json-in-place` is external in the Rollup config. The change happened in this 19c6c9547ff3dac0b3793074698ffe239c964d8f. 

I can fix it by manually editing the index.js. There's another line further down where I had to do the same thing for yargs_parser. 

```js
var inplace = require('json-in-place');
```

This works, but manually editing a file in node_modules isn't a solution. I could send a PR with the updated .js file, but it's just going to get broken again the next time a build is run. I'm guessing something needs to be changed in the Rollup or TypeScript config. 

In the meantime, this library seems to be working for others. Maybe I'm just doing something wrong?

### PR DESCRIPTION
**Pull request by pmcelhaney**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,6 +75,7 @@
   },
   "dependencies": {
     "json-in-place": "1.0.1",
+    "rollup-plugin-typescript": "^1.0.0",
```
Comment (markis) [inline package.json]: Could you move this to the `devDependency` section?

Comment (markis) [inline package.json]: Could you set the version to the exact version you're expecting?

File: rollup.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import typescript from '@markis/rollup-plugin-typescript';
+import typescript from 'rollup-plugin-typescript';
```
Comment (markis) [inline rollup.config.js]: This didn't used to work; great that it works now! :+1:

**Comment by pmcelhaney** [comment]
Done. Awesome tool. Thanks for sharing!

**Comment by markis** [comment]
jest-ratchet@1.3.2 has been published

**Pull request merged**